pallets/vector/src/weights.rs

```diff
@@ -206,6 +209,18 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 			.saturating_add(T::DbWeight::get().reads(5_u64))
 			.saturating_add(T::DbWeight::get().writes(1_u64))
 	}
+
+	fn set_function_ids() -> Weight {
```

Comment:
I will assume that we don't need to correct weights right now since we are going to regenerate everything in the next phase. It would be great if we could still test it and check if our benchmarks actually work.